patterns/template-query-loop-vertical-header-blog.php

```diff
@@ -44,7 +44,7 @@
 
 	<!-- wp:query-no-results -->
 		<!-- wp:paragraph -->
-		<p><?php echo esc_html_x( 'Sorry, but nothing was found. Please try a search with different keywords.', 'Message explaining that there are no results returned from a search', 'twentytwentyfive' ); ?></p>
```

Comment:
Matches the context to the more common version, with the period.
https://translate.wordpress.org/projects/wp-themes/twentytwentyfive/fr/default/?filters%5Bterm%5D=Sorry%2C+but+nothing+was+found